darts/tests/test_torch_forecasting_model.py

```diff
@@ -0,0 +1,127 @@
+from .base_test_class import DartsBaseTestClass
+from ..logging import get_logger
+from unittest.mock import patch
+from unittest import skip
+import tempfile
+import shutil
+import pandas as pd
+from ..timeseries import TimeSeries
+
+logger = get_logger(__name__)
+
+try:
+    from ..models.rnn_model import RNNModel
+    TORCH_AVAILABLE = True
+except ImportError:
+    logger.warning('Torch not available. RNN tests will be skipped.')
+    TORCH_AVAILABLE = False
+
+
+if TORCH_AVAILABLE:
+    class TestTorchForecastingModel(DartsBaseTestClass):
+
+        def setUp(self):
+            self.temp_work_dir = tempfile.mkdtemp(prefix='darts')
+
+        def tearDown(self):
+            shutil.rmtree(self.temp_work_dir)
+
+        def test_create_instance_new_model_no_name_set(self):
+            model = RNNModel('RNN', 10, 10, work_dir=self.temp_work_dir)
+            # no exception is raised
+            model2 = RNNModel('RNN', 10, 10, work_dir=self.temp_work_dir)
+            # no exception is raised
+
+        def test_create_instance_existing_model_with_name_no_fit(self):
+            model_name = 'test_model'
+            model1 = RNNModel('RNN', 10, 10, work_dir=self.temp_work_dir, model_name=model_name)
+            # no exception is raised
+
+            model2 = RNNModel('RNN', 10, 10, work_dir=self.temp_work_dir, model_name=model_name)
+            # no exception is raised
+
+        @patch('darts.models.torch_forecasting_model.TorchForecastingModel.reset_model')
```

Comment:
@pennfranc this and the next test case cover `reset_model()` - is it sufficient for you or do you want to cover another scenario?

Reply:
Ah yes sorry I missed this one, thanks!